hpaction/build_hpactionvars.py

```diff
@@ -70,22 +79,37 @@ def user_to_hpactionvars(user: JustfixUser) -> hp.HPActionVariables:
     # We'll be adding support for harassment and fee waiver later, but
     # for now we'll just support repairs.
     v.action_type_ms = [hp.ActionTypeMS.REPAIRS]
+    v.sue_for_repairs_tf = True
+    v.request_fee_waiver_tf = False
+    v.sue_for_harassment_tf = False
```

Comment:
It's weird to me that the action type is specified *both* as a multi-select field *and* as a series of boolean flags.  The former wasn't actually present in the XML I generated, so maybe it's just not being used anymore, but it _is_ still part of the schema in `Master.cmp`...